XenModel/Messages.resx

```diff
@@ -1209,6 +1221,24 @@
   <data name="ACTION_VM_TEMPLATIZING_TITLE" xml:space="preserve">
     <value>Converting VM '{0}' to template</value>
   </data>
+  <data name="ACTION_VUSB_CREATED" xml:space="preserve">
+    <value>USB device attached</value>
+  </data>
+  <data name="ACTION_VUSB_CREATING" xml:space="preserve">
+    <value>Attaching USB device '{0}' to VM '{1}'</value>
+  </data>
+  <data name="ACTION_VUSB_DELETED" xml:space="preserve">
+    <value>USB device deleted</value>

```

Comment:
Although we are internally deleting the VUSB, the message that the user needs to see is _detachig_. 
So:
ACTION_VUSB_DELETED: USB device detached
ACTION_VUSB_DELETING: Detaching USB device '{0}' from VM '{1}'
ACTION_VUSB_DETACH: Detach USB device
ACTION_VUSB_DETACH_CONFIRM: Are you sure you want to detach the selected USB device?

Reply:
Accepted.